File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Miniplayer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Miniplayer{

	@SerializedName("miniplayerRenderer")
	private MiniplayerRenderer miniplayerRenderer;

	public void setMiniplayerRenderer(MiniplayerRenderer miniplayerRenderer){
		this.miniplayerRenderer = miniplayerRenderer;
	}

	public MiniplayerRenderer getMiniplayerRenderer(){
		return miniplayerRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Miniplayer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Miniplayer : Parcelable, Serializable {
    @SerializedName("miniplayerRenderer")
    var miniplayerRenderer: MiniplayerRenderer? = null
    override fun toString(): String {
        return "Miniplayer{" +
                "miniplayerRenderer = '" + miniplayerRenderer + '\'' +
                "}"
    }
}